kotest-runner/kotest-runner-junit5/src/jvmMain/kotlin/io/kotest/runner/junit/platform/gradle/GradlePostDiscoveryFilterExtractor.kt

```diff
@@ -52,6 +59,12 @@ object GradlePostDiscoveryFilterExtractor {
       return field.get(obj) as List<Any>
    }
 
+   private fun buildScriptIncludePatterns(obj: Any): List<Any> {
+      val field = obj::class.java.getDeclaredField("buildScriptIncludePatterns")
```

Comment:
added support for filters defined in build script

Reply:
Are there build script exclude patterns as well? Not saying we need to fix it now, but could be flagged up as a potential future contribution